soh/src/overlays/misc/ovl_kaleido_scope/z_kaleido_map_PAL.c

```diff
@@ -86,9 +88,6 @@ void KaleidoScope_DrawDungeonMap(PlayState* play, GraphicsContext* gfxCtx) {
                     pauseCtx->cursorX[PAUSE_MAP] = 0;
                     pauseCtx->cursorPoint[PAUSE_MAP] = pauseCtx->dungeonMapSlot;
                     osSyncPrintf("kscope->cursor_point=%d\n", pauseCtx->cursorPoint[PAUSE_MAP]);
-                    R_MAP_TEX_INDEX =
```

Comment:
There is no longer an array of dungeon map textures. Each scene store it's own. R_MAP_TEX_INDEX is used to present what room link is in in some of the UI code, so it's removal has repercussions. Luckily, there are other things we can use.